### PR TITLE
doc: gsg: macOS: Do not require wget, use curl instead

### PR DESCRIPTION
**Pull request by carlescufi**
macOS ships with curl by default. There's no need to add additional tools to the installation when a perfectly good one ships with the OS.

### CODE REVIEW

**Review by kartben — CHANGES_REQUESTED** [review]
well spotted!

Please make sure https://github.com/zephyrproject-rtos/zephyr/blob/bb5098da082b3e5e959918a43a8810843754450c/doc/develop/toolchains/zephyr_sdk.rst#L133-L135 is also updated. .... but it feels like we may actually want to directly `..include` snippets from this file it in the getting started guide, looks like a lot of duplicated content (and I now remember on or the other of these two files not being properly updated during last update to 0.16.3).

There are a couple other occurrences of wget usage in the tree but I think it's fine to assume people getting that far would know how to install wget, if they haven't already.

**Comment by carlescufi** [comment]
> is also updated. .... but it feels like we may actually want to directly `..include` snippets from this file it in the getting started guide, looks like a lot of duplicated content (and I now remember on or the other of these two files not being properly updated during last update to 0.16.3).

OK, I will give this a go, not sure if it'll work out with tabs and so on?

**Comment by kartben** [comment]
> > is also updated. .... but it feels like we may actually want to directly `..literalinclude` snippets from this file it in the getting started guide, looks like a lot of duplicated content (and I now remember on or the other of these two files not being properly updated during last update to 0.16.3).
> 
> OK, I will give this a go, not sure if it'll work out with tabs and so on?

There's no reason why it shouldn't work, even in tab content, as long as nested indentation is respected. And you'll need the start-after/end-before markers to identify which portions to include of course

**Comment by kartben** [comment]
> > is also updated. .... but it feels like we may actually want to directly `..literalinclude` snippets from this file it in the getting started guide, looks like a lot of duplicated content (and I now remember on or the other of these two files not being properly updated during last update to 0.16.3).
> 
> OK, I will give this a go, not sure if it'll work out with tabs and so on?

There's no reason why it shouldn't work, even in tab content, as long as nested indentation is respected. And you'll need the start-after/end-before markers to identify which portions to include of course

**Review by canisLupus1313 — DISMISSED** [review]

**Comment by carlescufi** [comment]
> > > is also updated. .... but it feels like we may actually want to directly `..include` snippets from this file it in the getting started guide, looks like a lot of duplicated content (and I now remember on or the other of these two files not being properly updated during last update to 0.16.3).
> > 
> > 
> > OK, I will give this a go, not sure if it'll work out with tabs and so on?
> 
> There's no reason why it shouldn't work, even in tab content, as long as nested indentation is respected. And you'll need the start-after/end-before markers to identify which portions to include of course

I think it should work now @kartben. Please take another look.

File: doc/develop/toolchains/zephyr_sdk.rst [hunk 1]
```diff
@@ -67,153 +67,173 @@ the recommended version for the corresponding Zephyr version.
...
+***********************
+
+.. toolchain_zephyr_sdk_install_start
+
+.. note:: You can change ``0.16.3`` to another version in the instructions below
+          if needed; the `Zephyr SDK Releases`_ page contains all available
+          SDK releases.
+
+.. note:: If you want to uninstall the SDK, remove the directory where you installed it.
+          If you relocate the SDK directory, you need to re-run the setup script.
```
Comment (kartben) [inline doc/develop/toolchains/zephyr_sdk.rst:82]: this is redundant as already mentioned at the end of the install instructions? I think it should be removed from one or the other(s), but I don't have a strong opinion as to which

File: doc/develop/toolchains/zephyr_sdk.rst [hunk 2]
```diff
@@ -67,153 +67,173 @@ the recommended version for the corresponding Zephyr version.
...
+Zephyr SDK installation
+***********************
+
+.. toolchain_zephyr_sdk_install_start
+
+.. note:: You can change ``0.16.3`` to another version in the instructions below
+          if needed; the `Zephyr SDK Releases`_ page contains all available
+          SDK releases.
+
+.. note:: If you want to uninstall the SDK, remove the directory where you installed it.
```
Comment (kartben) [inline doc/develop/toolchains/zephyr_sdk.rst:81]:
```suggestion
.. note:: If you want to uninstall the SDK, you may simply remove the directory where you installed it.
```

**Review by kartben — CHANGES_REQUESTED** [review]
Sphinx (and negative diffs!) ftw!

File: doc/develop/toolchains/zephyr_sdk.rst (hunk 1 above)
Comment (carlescufi) [reply doc/develop/toolchains/zephyr_sdk.rst]: Fixed

File: doc/develop/toolchains/zephyr_sdk.rst (hunk 2 above)
Comment (carlescufi) [reply doc/develop/toolchains/zephyr_sdk.rst]: Thanks!

**Review by kartben — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Review by canisLupus1313 — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Comment by kartben** [comment]
@carlescufi looks like we should try doing something similar with "Install Dependencies" steps as we tend to get disconnects between what's in the getting started and what's in `installation_*.rst` files (see ex. #66461)

**Pull request merged**